2-mandatory/7-lane-names.js

```diff
@@ -6,7 +6,10 @@
   HINT: string and array methods that could be helpful (indexOf, filter)
 */
 
-function getLanes() {}
+function getLanes(streets) {
+  let newArr = streets.filter((street) => street.includes("Lane"));
+  return newArr;
+}
```

Comment:
Hi Natalie. I found your solution to mandatory 7 very helpful. I was struggling with the syntax, which is supposed to be very easy, but i just could not understand it. But yours made it very very easy for me to follow. Great job.